app/controllers/my_facilities/drug_stocks_controller.rb

```diff
@@ -0,0 +1,63 @@
+class MyFacilities::DrugStocksController < AdminController
+  include Pagination
+  include MyFacilitiesFiltering
+
+  layout "my_facilities"
+
+  before_action :authorize_my_facilities
+  after_action :verify_authorization_attempted
+  before_action :set_facility, only: [:new, :create]
+  before_action :drug_stocks_enabled?
+
+  def index
+    @facilities = filter_facilities()
+                    .includes(facility_group: :protocol_drugs)
+                    .where(protocol_drugs: {stock_tracked: true})
+  end
+
+  def new
+    render partial: "form"
+  end
+
+  def create
+    drug_stocks = DrugStock.transaction do
```

Comment:
Do we need a transaction here? If the create fails, it'll be silent anyway I believe, so there'd be nothing to rollback, it'll continue creating the others.

Reply:
We're showing a notice/alert based on the whole payload succeeding or failing. Better than failing silently.

Reply:
I actually wanted the transaction semantics. I'll change this to `create!`, ya?